src/sparseml/exporters/transforms/initializers_to_uint8.py

```diff
@@ -33,6 +37,7 @@ def transform(self, model: ModelProto) -> ModelProto:
             arr_int8 = numpy_helper.to_array(init)
             if arr_int8.dtype != numpy.int8:
                 continue
+            _LOGGER.debug(f"Matched {init.name}")
```

Comment:
yeah definitely thinking having a standard transform method to handle logging will be helpful